### ISSUE TITLE
Why is  { mutate }  not bound to the key by default?

### ISSUE THREAD

**Issue by joed1rk** [nandorojo/swr-firestore#11]
Hey me again :)

I noticed that `mutate` in this scenario
```javascript
const { mutate } = useDocument('/foo/bar')
```
is **not** bound to the `'/foo/bar'` document cache the way SWR binds it.
With SWR there is a global `mutate` that you can import from the lib which you have to pass a key to as first arg. But if you grab mutate from a specific useSWR invocation it is auto-bound to the cache key.

Just wondering if you had reasons for doing it this way :)

cheers

**Comment by nandorojo** [comment]
You're right – this is an unintentional bug. I can change it!

Keep in mind – the `useDocument` also exports a `set` and `update` function that **are** connected and **also** edit Firestore.

**Comment by joed1rk** [comment]
Right, I am aware, but I needed the mutate for specific reason.

Well it's totally you're call. But if you wanted to align as closely as possible with the useSWR API, it might be a good idea to follow that trend as well :)

Anyways, thanks again for building such a great tool

**Comment by nandorojo** [comment]
Fixed in `0.7.5`

```sh
yarn add @nandorojo/swr-firestore
```
